src/main/java/it/gov/pagopa/pu/debtpositions/service/massive/action/InsertionActionMassiveDebtPositionServiceImpl.java

```diff
@@ -0,0 +1,76 @@
+package it.gov.pagopa.pu.debtpositions.service.massive.action;
+
+import it.gov.pagopa.pu.debtpositions.dto.generated.*;
+import it.gov.pagopa.pu.debtpositions.exception.custom.ConflictErrorException;
+import it.gov.pagopa.pu.debtpositions.mapper.DebtPositionMapper;
+import it.gov.pagopa.pu.debtpositions.model.DebtPosition;
+import it.gov.pagopa.pu.debtpositions.model.InstallmentNoPII;
+import it.gov.pagopa.pu.debtpositions.repository.InstallmentNoPIIRepository;
+import it.gov.pagopa.pu.debtpositions.service.create.debtposition.CreateDebtPositionService;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Service;
+
+import java.util.Optional;
+import java.util.Set;
+
+@Service
+@Slf4j
+public class InsertionActionMassiveDebtPositionServiceImpl implements InsertionActionMassiveDebtPositionService {
```

Comment:
ps: when creating Service which we don't plan to use outside the current base package, we could avoid to create interfaces.

Interfaces must be created just for the entry point of a business logic, inner services could be created without
```suggestion
public class InsertInstallmentServiceImpl implements InsertionActionMassiveDebtPositionService {
```